docs/sources/shared/visualizations/panel-filtering.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Panel filtering
+---
+
+<!-- use heading: Panel filtering with ad hoc filters -->
+
+<!-- vale Grafana.WordList = NO -->
+<!-- vale Grafana.Spelling = NO -->
+
+In table visualizations, you can apply ad hoc filters from the visualization with one click.
+To quickly ad hoc filter variables, follow these steps:
```

Comment:
How about this?
```suggestion
To quickly filter the dashboard data, follow these steps:
```

We're not filtering variables here. We're filtering the dashboard itself (or the dashboard data themselves).